pysmt/optimization/goal.py

```diff
@@ -75,7 +74,10 @@ def get_logic(self):
             return logic
 
     def signed(self):
-        return True
+        return self.bv_signed
```

Comment:
Not a fan of getters and setters, but OK.

A better alternative would be to use the `@property` decorator: https://docs.python.org/3/library/functions.html#property